linode/networkreservedip/datasource_model.go

```diff
@@ -0,0 +1,54 @@
+package networkreservedip
```

Comment:
nitpick: Could the packages for these resources be renamed to `reservedip` and `reservedips` to match the resource names?

Reply:
Yeah that makes sense! I have now changed the names accordingly.